app/uk/gov/hmrc/iossreturns/connectors/FinancialDataHttpParser.scala

```diff
@@ -40,6 +40,9 @@ object FinancialDataHttpParser extends Logging {
         case NOT_FOUND =>
           logger.warn(s"Got not found from financial data $url")
           Right(None)
+        case SERVICE_UNAVAILABLE =>
```

Comment:
I feel like we should just combine all errors and handle them the same for this endpoint - we don't want to block the user journey if financial data is not replying properly

Can you combine all errors, IE https://github.com/hmrc/ioss-returns/pull/63/files#diff-3e8e28636d9984389a1b6ef8f4fe0fb15e0f4f935d52dc0a8e05b3d944b1a7f4R49 and make that reply with `None`, this also removes the need for an `Either` as well

Reply:
I've updated `getFinancialData` to return None for all errors